projects/canopy/src/lib/forms/radio/radio-group.component.ts

```diff
@@ -157,6 +152,14 @@ export class LgRadioGroupComponent implements ControlValueAccessor {
     }
   }
 
+  ngAfterContentInit(): void {
+    if (this.radios && this.stack) {
+      this.radios.toArray().forEach((radio) => {
```

Comment:
should radios be typed here `: LgRadioButtonComponent`? if not can remove the parenthesis?

Reply:
I think the type is unnecessary in this case as `this.radios` already implies it. 
Canopy has different lint rules so we currently keep the parenthesis across the codebase. I have a ticket to update the lint rules to match those of the monorepo 🙂